src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -60,3 +61,67 @@ def test_ipa_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvid
     status = ipa.sssctl.domain_status(trusted.domain, online=True)
     assert "online status: offline" not in status.stdout.lower(), "AD domain went offline!"
     assert "online status: online" in status.stdout.lower(), "AD domain was not online!"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.ticket(jira="RHEL-14752")
+@pytest.mark.topology(KnownTopologyGroup.AnyIPATrust)
+def test_ipa_trusts__ipa_master_lookup_trusted_user(ipa: IPA, trusted: IPA):
+    """
+    :title: Basic IPA-IPA Trust lookup on IPA server
+    :setup:
+        1. Restart SSSD and clear cache on IPA server
+    :steps:
+        1. Resolve trusted domain admin user
+    :expectedresults:
+        1. User is resolved
+    :customerscenario: True
+    """
+    ipa.sssd.restart(clean=True)
+
+    # Resolve user
+    username = trusted.admin_fqn
+    id_user = ipa.tools.id(username)
+
+    # Resolve group
+    groupname = trusted.fqn("admins")
+    getent_group = ipa.tools.getent.group(groupname)
+
+    assert id_user is not None, "Trusted admin user not found"
+    assert id_user.user.name == username, "Username does not match"
+
+    assert getent_group is not None
```

Comment:
@jakub-vavra-cz is the stickler for the messages, every assert. ```No group found!```

Reply:
Fixed.